packages/toaster/tests/Toaster.test.tsx

```diff
@@ -33,17 +33,19 @@ describe("Toaster", () => {
     expect(component.state("toasts")).not.toContain(testToast[0].props.id);
   });
 
-  it("dismisses toasts after specified time", () => {
-    const testToast = [<Toast id={0} title="I Am Toast" key={0} />];
+  it("calls onDismiss callback when toast is dismissed", () => {
+    const dismissCallback = jest.fn();
+    const testToast = [
+      <Toast id={0} title="I Am Toast" key={0} onDismiss={dismissCallback} />
+    ];
     const component = shallow(<Toaster children={testToast} />);
+    const instance = component.instance() as Toaster;
+    instance.dismissToast(testToast[0].props.id);
 
-    expect(Object.keys(component.state("toasts")).length).toBe(1);
-    setTimeout(() => {
-      expect(Object.keys(component.state("toasts")).length).toBe(0);
-    }, DELAY_TIME + 1);
+    expect(dismissCallback).toHaveBeenCalledTimes(1);
   });
 
-  it("dismisses autodismiss toasts after specified time", () => {
+  it("dismisses autodismiss toasts after specified time", done => {
```

Comment:
added done so that test worked. without this would always pass.

Reply:
Nice, I didn't know about that parameter